riscv/tests/riscv.rs

```diff
@@ -47,11 +47,7 @@ fn run_continuations_test(case: &str, powdr_asm: String) {
         .with_prover_inputs(Default::default())
         .with_output(tmp_dir.to_path_buf(), false);
     let pipeline_callback = |pipeline: Pipeline<GoldilocksField>| -> Result<(), ()> {
-        // Can't use `verify_pipeline`, because the pipeline was renamed in the middle of after
-        // computing the constants file.
```

Comment:
I don't really understand this comment, since what `verify_pipeline` does is **exactly** what the 3 lines below do.